WooCommerce/Classes/ViewRelated/Products/Edit Product/Downloadable Files/File List/ProductDownloadListViewController.swift

```diff
@@ -123,9 +119,26 @@ extension ProductDownloadListViewController {
         viewModel.completeUpdating(onCompletion: onCompletion)
     }
 
-    @objc private func addButtonTapped() {
-        // TODO: - add analytics
-        addEditDownloadableFile(indexPath: nil, formType: .add)
+    @objc private func addButtonTapped(_ sender: UIButton) {
+        // TODO: - add analytics M5
+
+        let title = Localization.bottomSheetTitle
+        let viewProperties = BottomSheetListSelectorViewProperties(title: title)
+        let actions = viewModel.bottomSheetActions
+        let dataSource = DownloadableFileBottomSheetListSelectorCommand(actions: actions) { [weak self] action in
+            self?.dismiss(animated: true) { [weak self] in
+                switch action {
+                case .fromDevice:
+                    return
+                case .fromWordPressMediaLibrary:
+                    return
+                case .fromFileURL:
+                    self?.addEditDownloadableFile(indexPath: nil, formType: .add)
+                }
+            }
+        }
+        let listSelectorPresenter = BottomSheetListSelectorPresenter(viewProperties: viewProperties, command: dataSource)
+        listSelectorPresenter.show(from: self, sourceView: sender, arrowDirections: .down)
```

Comment:
We might want to change the arrow direction to `.up` for iPad.

`.down` | `.up`
-- | --
![Simulator Screen Shot - iPad Air (3rd generation) - 2020-10-22 at 17 56 18](https://user-images.githubusercontent.com/1945542/96856473-868a7880-1490-11eb-91f9-683fe77d9b9b.png) | ![Simulator Screen Shot - iPad Air (3rd generation) - 2020-10-22 at 18 01 05](https://user-images.githubusercontent.com/1945542/96856538-9b670c00-1490-11eb-9fe1-8630c1333085.png)





```suggestion
        listSelectorPresenter.show(from: self, sourceView: sender, arrowDirections: .up)
```